src/freenet/clients/http/SecurityLevelsToadlet.java

```diff
@@ -686,6 +693,25 @@ public static void generatePasswordFormPage(boolean wasWrong, ToadletContainer c
 		HTMLNode form = ctx.addFormChild(content, postTo, "masterPasswordForm");
 		generatePasswordFormPage(wasWrong, form, content, forDowngrade, forUpgrade, physicalSecurityLevel, redirect);
 	}
+	
+	public static void generatePasswordConfirmationForm(HTMLNode formNode) {
```

Comment:
There's no need for that, it's a style issue (that does not affect readability), not a defect.

Don't fix it if it ain't broken, it will be addressed once that code sees other changes.